TimetableApp/TimetableApp.UWP/TimetableApp.UWP.csproj

```diff
@@ -32,8 +35,8 @@
     <AssemblyName>TimetableApp</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.18362.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.18362.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion>10.0.19041.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.19041.0</TargetPlatformMinVersion>
```

Comment:
We should not do this for compatibility with older Windows.

Reply:
thra em k hiểu cái này lắm, tự nhiên nó hiện trên github desktop í

Reply:
1. We should speak English :)
2. You seem to not have installed the UWP SDK for Windows 10 Version 18362. While this might not affect your build experience, you **should** exclude this file from your commit to prevent disturbing the main branch. And we would also want to maintain as much compatibility as possible.

Reply:
oh okay